src/app/auth/services/auth.service.ts

```diff
@@ -6,14 +6,17 @@ import { Location } from '@angular/common';
 import { Observable } from 'rxjs';
 import { pluck, mapTo, map } from 'rxjs/operators';
 
-import { ISignIn } from '../interfaces/signin.interface';
-import { ISignUp } from '../interfaces/signup.interface';
+import { ISignIn, ISignUp } from '../interfaces';
 
 @Injectable({
   providedIn: 'root',
 })
 export class AuthService {
 
+  public get isLogged(): boolean {
```

Comment:
Pretty strange that you have used getter and instance method together.

If you want to have getter isLogged than you have to remove isLogged() instance method. You don't need to have them both